autoendpoint/src/routers/mod.rs

```diff
@@ -169,8 +169,27 @@ impl From<RouterError> for ApcErrorKind {
         match err {
             RouterError::Adm(e) => ApcErrorKind::EndpointError("Router:Adm", e.to_string()),
             RouterError::Apns(e) => ApcErrorKind::EndpointError("Router:APNS", e.to_string()),
-            RouterError::Fcm(e) => ApcErrorKind::EndpointError("Router:FCM", e.to_string()),
-            RouterError::TooMuchData(e) => ApcErrorKind::EndpointError("TooMucData", e.to_string()),
+            RouterError::Fcm(e) => {
```

Comment:
This `From` impl currently isn't being used.

I'm still of the opinion that autoendpoint should have its own high level error type that's more specific to its crate. So IMO `ApcError` should only be used by autoendpoint as Errors triggered from the common crate (that are probably ultimately wrapped by one of its own errors) and thus converting from the inverse (endpoint specific to common) like this wouldn't be needed.

Reply:
That's fine, I'll remove the error bridge stuff. I still would prefer to have a single error class to avoid some duplication, but I'm not going to die on that hill. 

I'm not going to do it in this PR, but we ought to change `ApiError` to be `ApeError` to mimic `ApcError`. `ApiError` is legacy as well and no longer refers to anything meaningful.